### PR TITLE
LAB3 - KubeVirt Upgrades

### PR DESCRIPTION
**Pull request by iranzo**
Add LAB for KubeVirt Upgrades

https://deploy-preview-419--kubevirtio.netlify.com/labs/kubernetes/lab3.html

### CODE REVIEW

File: labs/kubernetes/lab3.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+~~~sh
+kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml
+~~~
+
+Let's check the deployment:
+~~~sh
+kubectl get pods -n kubevirt
+~~~
+
+Once it's ready, it will show something similar to the information below (this will keep showing in the upper half of the terminal in the right side of the webpage):
```
Comment (ptrnull) [inline labs/kubernetes/lab3.md]: I don't see the meaning of an upper half of the terminal in this lab, there is no tmux, isn't it?

File: labs/kubernetes/lab3.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+~~~
+
+Notice from the output that the VM is not running yet.
+
+To start a VM, `virtctl`~~~` should be used:
+
+~~~sh
+./virtctl start testvm
+~~~
+
```
Comment (ptrnull) [inline labs/kubernetes/lab3.md]: output missing

File: labs/kubernetes/lab3.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+Now you can check again the VM status:
+
+~~~sh
+kubectl get vms
+~~~
+
```
Comment (ptrnull) [inline labs/kubernetes/lab3.md]: output missing

File: labs/kubernetes/lab3.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+Now let's deploy KubeVirt by creating a Custom Resource that will trigger the 'operator' and perform the deployment:
+
+~~~sh
+kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml
+~~~
+
+Let's check the deployment:
+~~~sh
+kubectl get pods -n kubevirt
```
[the remark this answers is not in the document]
Comment (iranzo) [reply labs/kubernetes/lab3.md]: Shown in line after that one

**Comment by iranzo** [comment]
Added outputs requested @ptrnull

**Pull request merged**